src/docs/guides/27-how-to-handle-provider-onboarding-and-identity-verification/index.md

```diff
@@ -0,0 +1,230 @@
+  expired
+
+  ![fail view](verificationFailed.png)
+
+### 5. Updating provider Stripe account
+
+Most of the information related to Stripe account like email or address
+can be updated from Stripe's Connect onboarding. For updating this
+information we need to
+[create a new Account Link](https://www.sharetribe.com/api-reference/marketplace.html#create-stripe-account-link)
+and redirect user back to Connect onboarding.
+
+Only thing we manage on the FTW side is updating the `bankAccountToken`
+of the Stripe account. This means that if the provider want's to update
+their bank account number (e.g. IBAN), it's handled by passing a new
+bankAccountToken to
+[update Stripe account](https://www.sharetribe.com/api-reference/marketplace.html#update-stripe-account)
+API endpoint.
+
+**Note:** Currently, Stripe doesn't support updating the counrty of the
+account after the account has been created.
+
+## Using custom flow
+
+It's also possible to implement the onboarding flow in your own
+application. This way the user will stay on your application through the
+whole onboarding. The downside with this approach is that you are
+responsible for collecting all the required information and keeping the
+UI up-to-date also with the possible future changes.
+
+Before FTW-daily v4.0.0 and FTW-hourly v5.0.0 the Stripe onboarding was
+implemented like this also in FTW. There are some now deprecated
+components you can use as a starting point if you want to implement your
+own flow. You should keep in mind that these components will not be
+updated by our team since FTW will use Connect Onboarding by default.
+
+### Collecting required information
+
+When onboarding Custom accounts, you need to collect the required
+information for each account (which Stripe verifies). The information
+you need to collect depends on factors such as the capabilities and the
+country the account is based in. You can find the requirements from
+Stripe's
+[Required verification information](https://stripe.com/docs/connect/required-verification-information)
+documentation. You should keep in mind that depending on the situation,
+you might need to collect a scan of an ID document, an address document,
+or both to enable payouts so you need to implement way to download these
+files.
+
+After you have collected the information you need to obtain a new
+[account token](https://stripe.com/docs/connect/account-tokens) via
+Stripe API, through the same Stripe platform account as the one used by
+the marketplace. This account token should be passed to Flex API when
+creating or updating the Stripe account.
+
+### Updating the Stripe account
+
+Before FTW-daily v4.0.0 and FTW-hourly v5.0.0 it was not possible to
+update the Stripe account through the user's account settings. However,
+if you decide to use your own custom flow you should concider
+implementing the way to check the status of Stripe account and
+collecting more information about the user when it's required.
```

Comment:
If the _implementing a way to_ refers to _collecting_ here I would switch it to _collect_.